Sources/MapboxMaps/Location/LocationManager.swift

```diff
@@ -18,23 +18,24 @@ public class LocationManager: NSObject {
     /// The object that acts as the delegate of the location manager.
     public weak var delegate: LocationPermissionsDelegate?
 
-    /// Making variable `public private(set)` to have direct access to auth functions
+    /// Can be used to access location authorization settings.
```

Comment:
I think we should change this comment to `/// Property that provide location and authorization updates`